cmd/check.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func setupCheckCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "check",
+		Short: "Check the integration",
```

Comment:
Sorry for hijacking this PR but I think we could make a more consistent use of the terms `package` and `integration`. 

I understand that at this point in time `integration`s represent the only available `type` of package but this will hopefully change in the future.

So we should decide whether this tool is somehow dedicated to the development of specific packages of `type: integration`, and in this case I would change the name of the tool and maybe move the source code into the `elastic/integrations` repo; or we say this is a general purpose tool to support the development of packages, and in this case I'd consolidate the naming scheme around source code, docs, etc. avoiding to refer to `integration`s when the command can actually work with any package type.

Reply:
Hey @masci, thanks for looking into this! I believe we have a discussion about this in the past.

We decided to move on with packages, but as long as there are no other types even defined (or in plans), I sticked to integrations. I can try to get rid of words referring to "integrations" in as many places as possible, but internally there will be support only for integrations.

I prefer to keep this in a separate repository as the original plan is to keep integration source in the integrations repo and nothing else (no logic, tooling, side scripting).

I'd like to listen to other opinions on this. @exekias @ruflin @ycombinator

Reply:
I'm +1 on having this tool as the reference for package development, where integrations will be only a subset of that. In that sense I would refrain from mentioning "integrations" here.

In that sense I think it also makes sense to have the tool in a different repo, not only because of this "generic" soul, but also because I expect that once things get stable, `elastic-package` users won't necessarily be go developers (with the go toolchain installed for instance). This project should have its own lifecycle.

Reply:
If you don't mind I would go over this in another PR. It's not directly in the scope of this PR.

Reply:
https://github.com/elastic/elastic-package/issues/35